src/event_form/components/organizer/event-form-organizer-modal.controller.spec.js

```diff
@@ -335,4 +342,14 @@ describe('Controller: Event Form Organizer Modal', function() {
     expect($scope.newOrganizer.id).not.toEqual('73dbf765-1622-4454-aed9-6c1fe5771568');
     expect($uibModalInstance.close).not.toHaveBeenCalled();
   });
+
+  it('should verify if contact url is valid according to the regex', function(){
```

Comment:
This tests the regex, but not the usage of the regex in the component. Is there a way to either actually test the `ng-pattern` attribute used in the component, or if that's not possible move this test to a separate spec to specifically test the regex itself (and not duplicate this in both component specs)?